doc/dev/release-process.rst

```diff
@@ -210,7 +210,7 @@ See `the Ceph Tracker wiki page that explains how to write the release notes <ht
 
    .. prompt:: bash $
 
-      sync-push ceph octopus
+      sync-push ceph octopus-15.2.7
```

Comment:
Maybe time to update the example to a more recent release?

Reply:
I thought about that myself, done :)